lib/graphql-error-handler.js

```diff
@@ -0,0 +1,17 @@
+let graphqlErrorHandler = null;
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
+  const newrelic = require('newrelic');
+  graphqlErrorHandler = error => {
+    newrelic.noticeError(error, { graphql: true });
```

Comment:
It would, so if you want to retain those original error names than leave it as is. Just noting that NewRelic's UI won't actually take that `{ graphql: true }` property into consideration...

![image](https://cloud.githubusercontent.com/assets/555859/18889931/2b532572-84cd-11e6-8197-fb73327b672d.png)

Hence the error name hack to work around the NR limitation.